data/mods/MindOverMatter/powers/biokinesis.json

```diff
@@ -65,9 +60,17 @@
         "( (u_val('spell_level', 'spell: biokin_overcome_pain') * 2000) + 130000) * (scaling_factor(u_val('intelligence') ) )"
```

Comment:
this part is something i wondered often: do you plan to implement some another scaling factors? because if not, you can simplify it by moving `u_val('intelligence')` into math, and make the math accept zero arguments, transforming `scaling_factor(u_val('intelligence') )` into `scaling_factor()` that would work as overall multiplier

Reply:
I don't currently have plans to do that, but I did deliberately leave that out of the equation in case I did want to call it for something else (since "+/-5% per stat point centered around 10" seems like a nice way to do things)